packages/angular/build/tools/baseline_browserslist/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+load("@aspect_rules_js//js:defs.bzl", "js_binary")
+load("@aspect_rules_ts//ts:defs.bzl", "ts_config")
+load("//tools:defaults2.bzl", "jasmine_test", "ts_project")
+
+js_binary(
+    name = "baseline_browserslist",
+    data = [":baseline_browserslist_lib"],
+    entry_point = "index.mjs",
+    visibility = ["//packages/angular/build:__subpackages__"],
+)
+
+ts_project(
+    name = "baseline_browserslist_lib",
+    srcs = ["index.mts"],
+    source_map = True,
+    tsconfig = ":tsconfig",
+    deps = [":generate_browserslist"],
+)
+
+ts_project(
+    name = "generate_browserslist",
+    srcs = ["generate_browserslist.mts"],
+    source_map = True,
+    tsconfig = ":tsconfig",
+    deps = [
+        "//packages/angular/build:node_modules/baseline-browser-mapping",
+    ],
+)
+
+ts_project(
+    name = "generate_browserslist_test_lib",
+    testonly = True,
+    srcs = ["generate_browserslist_spec.mts"],
+    tsconfig = "//:test-tsconfig-esm",
+    deps = [
+        ":generate_browserslist",
+        "//:node_modules/@types/jasmine",
```

Comment:
Redundant

```suggestion
```

Reply:
Huh, why _isn't_ this necessary? I don't see anything which implicitly adds `//:node_modules/@types/jasmine` as a dependency...

Reply:
it's the tsconfig

Reply:
Interesting. Personally I think the `ts_project` _should_ have a dependency on Jasmine given that it contains the actual `describe`, `it`, etc. which actually relies on Jasmine typings. I feel like leaking the `tsconfig` deps into its targets is a strict deps violation.

Not super important for something as simple as this, just an observation.